.github/CODEOWNERS

```
@@ -158,10 +157,12 @@ Include/internal/pycore_time.h  @pganssle @abalkin
 /Tools/cases_generator/        @markshannon
 
 # AST
-Python/ast.c                  @isidentical @JelleZijlstra
-Parser/asdl.py                @isidentical @JelleZijlstra
-Parser/asdl_c.py              @isidentical @JelleZijlstra
-Lib/ast.py                    @isidentical @JelleZijlstra
+Python/ast.c                  @isidentical @JelleZijlstra @eclips4
+Python/ast_opt.c              @isidentical @eclips4
```

For consistency should @JelleZijlstra be added here to? 

```suggestion
Python/ast_opt.c              @isidentical @JelleZijlstra @eclips4
```

---

I guess when he added himself as an`ast` codeowner, he just forgot about the `ast_opt` file.
But let's wait for his opinion.

---

I didn't forget, I'm less interested in that file.